testutil-server/src/main/java/io/spine/testing/server/blackbox/BlackBoxContext.java

```diff
@@ -492,58 +505,130 @@ public void close() {
 
     /**
      * Obtains a Subject for an entity of the passed class with the given ID.
+     *
+     * @deprecated please use {@link #assertEntity(Object, Class)}
      */
-    public <I, E extends Entity<I, ? extends EntityState>>
+    @Deprecated
+    public final <I, E extends Entity<I, ? extends EntityState>>
     EntitySubject assertEntity(Class<E> entityClass, I id) {
-        @Nullable Entity<I, ?> found = findEntity(entityClass, id);
+        return assertEntity(id, entityClass);
+    }
+
+    /**
+     * Obtains a Subject for an entity of the passed class with the given ID.
+     */
+    public final <I, E extends Entity<I, ? extends EntityState>>
+    EntitySubject assertEntity(I id, Class<E> entityClass) {
```

Comment:
Why do we change the order of parameters? One could argue that specifying an entity type first and then its ID is more natural from the user's perspective.

Reply:
I mentioned this in one of the commit comments. Here's why. 1) When we assert against an entity state, it's a big block of code. Having the ID parameter below this block makes it poorly visible.

2) A reference to an object usually comes first in a procedure that deals with an object. We use IDs as reference to entities.